src/modules/google.js

```diff
@@ -87,6 +82,23 @@
 
 			// Map POST requests
 			post: {
+				// See: https://developers.google.com/photos/library/reference/rest/v1/mediaItems/search
+				'me/album': function(p, callback) {
+					p.data = {
+						pageSize: p.data.limit,
+						albumId: p.data.id
+					};
+					callback(mediaItemsSearchUrl);
+				},
+
+				// See: https://developers.google.com/photos/library/reference/rest/v1/mediaItems/search
+				'me/photo': function(p, callback) {
```

Comment:
This looks like a duplicate of the one above, please reference a function like the others. Thanks